helpers/calendar/analytics_helper.py

```diff
@@ -108,6 +108,11 @@ def get_event_details(self, event, calendar_id):
                     event_details["roomName"] = resource.get(
                         'displayName') or None
                     event_details["summary"] = event.get("summary")
+        elif event.get('organizer') and event.get(
+                'organizer').get('email') == calendar_id:
+            event_details["roomName"] = event.get(
+                'organizer').get('displayName') or None
+            event_details["summary"] = event.get("summary")
```

Comment:
This code was added to also get the event details of those booked from the app. Previously, it was only getting the event details of those booked from the calendar.